src/app/components/ThemeByNetwork/index.tsx

```diff
@@ -14,6 +15,10 @@ export const ThemeByNetwork: FC<{ network: Network; children: React.ReactNode }>
   </ThemeProvider>
 )
 
-export const withDefaultTheme = (node: ReactNode) => (
+export const withMainnetTheme = (node: ReactNode) => (
   <ThemeByNetwork network={Network.mainnet}>{node}</ThemeByNetwork>
 )
+
+export const withDefaultTheme = (node: ReactNode) => (
+  <ThemeByNetwork network={fixedNetwork ?? Network.mainnet}>{node}</ThemeByNetwork>
+)
```

Comment:
Still not sure why we need 2 Themes? And are we making sure, we are not accidentally including both?

Reply:
- `withMainnetTheme()` is only ever used for the external part of the `HomePage` (ie. opening graph). We must use it there, because the `HomePage` is not equipped to be displayed with the `TestTheme`.  (We would get black text on dark blue bg, etc.)
- However, the internal part of the `HomePage` (which shows the actual graph) will still get a theme corresponding to the network currently selected.
- The other wrapper, `withDefaultTheme()` is being used in other locations, when we can use any theme corresponding to the (default) network.

The reason we need 2 wrappers now, when we could do with 1 earlier, is that now  the default network might be testnet, while earlier it was always mainnet, and so when using `withDefaultTheme`, we might get the testnet theme as the default theme, which is not suitable for everywhere.

Please see the demos about how this works in practice.

Reply:
Ok, still think that a simple if condition could fix this. And we could just use one theme. Or the HOC could accept additional params, to make the theme fixed, in the `HomePage` case. Either way, I guess it's fine.

Reply:
>  Or the HOC could accept additional params, to make the theme fixed, in the HomePage case.

Did that now. How do you like it now?